components/Carousel/Carousel.js

```diff
@@ -1,8 +1,28 @@
 class Carousel {
+    constructor(carousel) {
+        this.carousel = carousel;
+        this.images = this.carousel.querySelectorAll("img")
+        this.rButton = this.carousel.querySelector(".right-button");
+        this.lButton = this.carousel.querySelector(".left-button");
+        this.currentIndex = 0;
+        currentImage = this.images[this.currentIndex].style.display = "block";
+        
+        this.rButton.addEventListener("click", () => this.toggleRight());
+        this.lButton.addEventListener("click", () => this.toggleLeft());
+    }
 
+    toggleRight() {
+
+    }
+
+    toggleLeft() {
+
+    }
 }
 
-let carousel = document.querySelector();
+let carousel = document.querySelectorAll(".carousel");
+
+carousel.forEach(carousel => new Carousel(carousel));
```

Comment:
`forEach` not required here since we only have a single element with the class `.carousel`:

```
let carousel = new Carousel(document.querySelector('.carousel'));
```